.github/workflows/upload_alerts.yml

```diff
@@ -0,0 +1,38 @@
+# upload alerts every 15 minutes
+
+name: Upload Alerts to AWS/Rockset
+
+on:
+  schedule:
+    - cron: '*/15 * * * *'
+
+jobs:
+  upload-alerts:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Install Python Packages
+        run: |
+          pip3 install rockset==1.0.3
+          pip3 install boto3==1.19.12
+
+      - name: Create alerts
+        run: |
+          output=$("python tools/alerts/create_alerts.py")
+          echo "uploading following alerts"
+          echo "$output"
+          echo "::set-output name=script_output::$output"
```

Comment:
He he, this will be deprecated soon, you should use the new GITHUB_OUTPUT file.  Luckily, there is one issue this sprint exactly about this issue https://github.com/pytorch/pytorch/issues/102550